step.yml

```diff
@@ -0,0 +1,67 @@
+title: Android Instrument Test
+summary: Runs Instrument tests on an existing APK
+description: Runs Instrument tests on an existing APK
+website: https://github.com/hisaac/bitrise-step-android-instrument-test
+source_code_url: https://github.com/hisaac/bitrise-step-android-instrument-test
+support_url: https://github.com/hisaac/bitrise-step-android-instrument-test/issues
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+
+project_type_tags:
+- android
+
+type_tags:
+- test
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+deps:
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+
```

Comment:
unnecessary newline, one newline is enough to separate property blocks

Reply:
Works for me. Fixed in d3f69ab

Reply:
Nice, approved the PR!